### PR TITLE
Fix: PestAPI 

### PR DESCRIPTION
**Pull request by ItsEmpa**
## What
Adds support for pests in uncleaned plots, and other fixes regarding fixPests()

## Changelog Fixes
+ Fixed incorrect pest amount in plots under certain conditions. - Empa
+ Fixed pests in uncleared plots not getting detected. - Empa

### CODE REVIEW

File: src/main/java/at/hannibal2/skyhanni/features/garden/pests/PestAPI.kt [hunk 1]
```diff
@@ -88,11 +90,13 @@ object PestAPI {
                 plot.pests = 1
                 plot.isPestCountInaccurate = false
             }
+            return
```
Comment (hannibal002) [inline src/main/java/at/hannibal2/skyhanni/features/garden/pests/PestAPI.kt]: use `else if` instead

File: src/main/java/at/hannibal2/skyhanni/features/garden/pests/PestAPI.kt [hunk 2]
```diff
@@ -88,11 +90,13 @@ object PestAPI {
...
         }
         if (inaccurateAmount == 1) { // if we can assume all the inaccurate pests are in the only inaccurate plot
             val plot = getPlotsWithInaccuratePests().firstOrNull()
             plot?.pests = scoreboardPests - accurateAmount
             plot?.isPestCountInaccurate = false
+            return
```
Comment (hannibal002) [inline src/main/java/at/hannibal2/skyhanni/features/garden/pests/PestAPI.kt]: return at the end of a function can be ignored

**Review by hannibal002 — CHANGES_REQUESTED** [review]
code cleanup. logic looks fine

**Comment by ItsEmpa** [comment]
done

**Review by hannibal002 — APPROVED** [review]

**Pull request merged**